.conda/meta.yaml

```diff
@@ -1,43 +1,35 @@
 package:
   name: rbc
-  version: 0.1.0.dev2
+  version: 0.8.0
```

Comment:
I don't think this file is used anywhere, the feedstock has its own meta file. Perhaps we should just remove it.

Reply:
It's only used in some release scripts but if you are never running these we can remove this and the scripts.

Reply:
Is the script you mentioned `update-conda-pkg.sh`? If yes, the `meta.yaml` file it reads is the one from the `rbc-feedstock` repo. Also, I think we can remove this script as well since the conda release process is automatically triggered.